modules/telemetry/telemetry/BUILD

```diff
@@ -35,3 +36,31 @@ heph_cc_test(
         ":telemetry",
     ],
 )
+
+heph_cc_test(
+    name = "log_tests",
+    srcs = ["tests/log_tests.cpp"],
+    deps = [
+        ":telemetry",
+    ],
+)
+
+############
```

Comment:
Very nit. but I wouild put binaries before tests, so that it's easier to see them